multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -441,7 +441,7 @@ def transform_prediction(
         Predicted labels ready to compute metric scores.
         """
         if loss_func is not None and isinstance(loss_func, torch.nn.BCEWithLogitsLoss):
-            y_pred = torch.sigmoid(y_pred)
+            y_pred = torch.sigmoid(y_pred.float())
```

Comment:
We should revise the logic and add sigmoid in https://github.com/awslabs/autogluon/blob/bf995273bd16ed663b75e583feaf64f80b27e03c/multimodal/src/autogluon/multimodal/optimization/lit_module.py#L247